website/siteConfig.js

```diff
@@ -50,8 +50,8 @@ const siteConfig = {
   users,
 
   /* path to images for header/footer */
-  headerIcon: 'img/favicon.ico',
-  footerIcon: 'img/favicon.ico',
+  headerIcon: 'img/logo_icon.svg',
+  footerIcon: 'img/logo_icon.svg',
   favicon: 'img/favicon.ico',
```

Comment:
Thanks for adding it, @mohamednajiub !
Can you add the `Favicon` here as well?

Reply:
I replaced the old favicon with the new favicon and kept for its name in the file so if you opened the favicon file you can see the new favicon

Reply:
I reviewed the file favicon.ico and it is the new favicon.